packages/core/src/utils/styleUtils/ssrStylesPattern.spec.ts

```diff
@@ -766,15 +765,7 @@ describe('pattern component', () => {
           },
         },
       },
-      usedComponents: [
-        {
-          sys: {
-            type: 'Link',
-            linkType: 'Entry',
-            id: patternNode.definitionId,
-          },
-        },
-      ],
+      usedComponents: [patternEntry],
```

Comment:
shouldn't it be a `Link`?

Reply:
It can be an array `Link` or a pattern entry https://github.com/contentful/experience-builder/blob/development/packages/core/src/types.ts#L225